python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -59,67 +52,87 @@ def core_info(engine):
     """
     import sgtk
     from sgtk.commands.core_upgrade import TankCoreUpdater
-    logger = sgtk.LogManager.get_logger(LOGGER_NAME)
-
-    code_css_block = "display: block; padding: 0.5em 1em; border: 1px solid #bebab0; background: #faf8f0;"
 
     # Create an upgrader instance that we can query if the install is up to date.
     installer = TankCoreUpdater(
         engine.sgtk.pipeline_configuration.get_install_location(),
-        logger,
+        engine._log,
```

Comment:
interesting. firstly, definitely add a comment explaining that we cannot use `engine.logger` because we have to support back to 0.16 cores. yes, i agree `engine.sgtk.log` is the way to go - but again, please add a comment explaining that this `sgtk.log` property is a special thing that only exists for the shotgun engine etc.